e2e/tests/api-driven/src/hasuraTriggers/helpers.ts

```diff
@@ -1,6 +1,48 @@
+import { TEST_EMAIL } from "../../../ui-driven/src/helpers/globalHelpers";
 import { $admin } from "../client";
+import { safely } from "../globalHelpers";
+import gql from "graphql-tag";
 
 export const cleanup = async () => {
   await $admin.user._destroyAll();
   await $admin.team._destroyAll();
 };
+
+export async function createDemoUser(demoTeamId: number) {
+  const variables = {
+    firstName: "Test",
+    lastName: "Test",
+    email: TEST_EMAIL,
+    teamId: demoTeamId,
+    role: "demoUser",
+  };
+
+  const response = await safely(() =>
+    $admin.client.request<{ insertUsersOne: { id: number } }>(
+      gql`
+        mutation CreateAndAddUserToTeam(
```

Comment:
This matches the query on the frontend - might be a good reason to wrap up on `planx-core`?